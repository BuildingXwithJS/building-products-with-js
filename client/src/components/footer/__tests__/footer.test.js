import Footer from '../index';

test('# Footer', () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper).toMatchSnapshot();
});
