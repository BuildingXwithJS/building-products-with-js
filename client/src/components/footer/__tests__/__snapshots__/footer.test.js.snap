exports[`test # Footer 1`] = `
<footer
  style={
    Object {
      "bottom": "0",
      "position": "absolute",
      "width": "100%",
    }
  }>
  <div
    className="row">
    <div
      className="col-xs-11 col-sm-9 col-md-7">
      <Connect(Notifications) />
    </div>
  </div>
</footer>
`;
